WinUIGallery/Navigation/NavigationRootPage.xaml

```diff
@@ -67,6 +67,7 @@
                     VerticalAlignment="Center"
                     Style="{StaticResource CaptionTextBlockStyle}"
                     Text="{x:Bind AppTitleText}" />
+                <TextBox x:Name="AppTitleBarTextBox" Width="300" Height="40" Margin="16,0,0,0" Visibility="Collapsed" /> 
```

Comment:
>300 [](http://example.com/codeflow?start=60&length=3)

What is the minimum width of the window? Does Width="300" cause problems if the window is resized to be narrow?

Reply:
The minimum window width is zero. Which, by itself, is already bad.

Reply:
i am using minwidth for now. 300 just looks big enough